base-template/about-base-template.md

```diff
@@ -1,45 +1,63 @@
-# The Base Template
+# About Base Template
 
-## How do I create a new template?
+## Introduction
 
```

Comment:
I feel documents should all start with: "This document is .... It is designed to be used by template authors to ...
(You have part of this content further in.